Specs/Scene/GltfLoaderSpec.js

```diff
@@ -868,6 +868,8 @@ describe(
         expect(featureIdTexture).toBeInstanceOf(
```

Comment:
Add a unit test for the large feature ID texture, so we have an example that includes a null feature ID